Tests/Unit/Microsoft365DSC/Microsoft365DSC.IntuneAccountProtectionPolicy.ps1

```diff
@@ -0,0 +1,319 @@
+[CmdletBinding()]
```

Comment:
This file should be named Microsoft365DSC.IntuneAccountProtectionPolicy.Tests.ps1. Currently the .Tests is missing at the end of the file.